packages/vitest/src/node/core.ts

```diff
@@ -101,7 +103,9 @@ export class Vitest {
       })
     }
 
-    this.reporters = await createReporters(resolved.reporters, this.runner)
+    this.reporters = resolved.benchmark
```

Comment:
shouldn't this be decided based on `mode`?

Reply:
Nice catch!

Reply:
I don't think `resolved.mode` is the same as VitestMode (`'test' | 'benchmark'`), it's custom user mode (`--mode`)

Reply:
We now have three modes 😄 

- `'WATCH' | 'RUN'`
- `'test' | 'benchmark'`
- custom, default `test` - btw, should we change it to `benchmark`, if Vitest is running as `vitest bench`?

Reply:
I see that now mode is always `'test' | 'benchmark'`, but it's still confusing, we will need to come up with better naming, I think :S

Still, I think we should pass down `benchmark` to `defineConfig` 🤔